src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,24 @@ const Cashier = (() => {
         });
     };
 
+    const setP2PVisibility = async () => {
+        const is_agent = !(await BinarySocket.send({ p2p_agent_info: 1 })).error;
```

Comment:
should we also check `is_mobile` here?

Reply:
in this case, we won't show p2p at all on mobile?

Reply:
no, in cashier page we won't show p2p at all, and in p2p page we will show a message in case someone goes on that page by direct link

Reply:
then in that case we will need to add again when p2p mobile is ready

Reply:
yes, that's why I asked to add todo comments :)